packages/engine/Source/Core/ITwin.js

```diff
@@ -0,0 +1,306 @@
+ *
+ * @see IonResource
+ * @see IonImageryProvider
+ * @see IonGeocoderService
+ * @see createWorldImagery
+ * @see createWorldTerrain
+ * @namespace ITwin
+ */
+const ITwin = {};
+
+/**
+ * Gets or sets the default iTwin access token.
+ *
+ * TODO: I'm not sure we can even do this kind of access token. Each route seems to need it's own scopes
+ * and we may not be able to guarantee this "top level token" has them all
+ * So far we use
+ * `mesh-export:read` for loading meshes GET /mesh-export(s)
+ * `mesh-export:modify` if we want to include a function to create an export
+ * `itwin-platform` if we want to use the iModel shares ourselves  GET /imodels/{id}/shares
+ *
+ *
+ * @type {string|undefined}
+ */
+ITwin.defaultAccessToken = undefined;
+
+/**
+ * Gets or sets the default iTwin API endpoint.
+ *
+ * @type {string|Resource}
+ * @default https://api.bentley.com
+ */
+ITwin.apiEndpoint = new Resource({
+  url: "https://api.bentley.com",
+});
+
+/**
+ * @param {string} exportId
+ */
+ITwin.getExport = async function (exportId) {
+  if (!defined(ITwin.defaultAccessToken)) {
+    throw new DeveloperError("Must set ITwin.defaultAccessToken first");
+  }
+
+  const headers = {
+    Authorization: ITwin.defaultAccessToken,
+    Accept: "application/vnd.bentley.itwin-platform.v1+json",
+  };
+
+  // obtain export for specified export id
+  const url = `${ITwin.apiEndpoint}mesh-export/${exportId}`;
+
+  // TODO: this request is _really_ slow, like 7 whole second alone for me
+  // Arun said this was kinda normal but to keep track of the `x-correlation-id` of any that take EXTRA long
+  const response = await fetch(url, { headers });
+  if (!response.ok) {
+    const result = await response.json();
+    if (response.status === 401) {
+      throw new RuntimeError(
+        `Unauthorized, bad token, wrong scopes or headers bad. ${result.error.details[0].code}`,
+      );
+    } else if (response.status === 404) {
+      throw new RuntimeError(`Requested export is not available ${exportId}`);
+    } else if (response.status === 429) {
+      throw new RuntimeError("Too many requests");
+    }
+    throw new RuntimeError(`Unknown request failure ${response.status}`);
+  }
+
+  /** @type {ExportResponse} */
+  const result = await response.json();
+  return result;
+};
+
+/**
+ * Get the list of exports for the given iModel + changeset
+ *
+ * @param {string} iModelId
+ * @param {string} changesetId
+ */
+ITwin.getExports = async function (iModelId, changesetId) {
+  if (!defined(ITwin.defaultAccessToken)) {
+    throw new DeveloperError("Must set ITwin.defaultAccessToken first");
+  }
+
+  const headers = {
+    Authorization: ITwin.defaultAccessToken,
+    Accept: "application/vnd.bentley.itwin-platform.v1+json",
+    Prefer: "return=representation", // or return=minimal (the default)
+  };
+
+  // obtain export for specified export id
+  let url = `${ITwin.apiEndpoint}mesh-export/?iModelId=${iModelId}`;
```

Comment:
I'd recommend to use `&exportType=3DTILES` to filter out other export types, and change the exportType enum above to 3DTILES too.
Also, you can do `&$top=5` to get the latest 5 exports, for performance reasons.

Reply:
@danieliborra I was hoping for a way to filter based on type but none are [documented](https://developer.bentley.com/apis/mesh-export/operations/get-exports/) so I didn't think it was possible. Are there other properties filterable like this?

Reply:
You are right. Since the API is in beta/tech. preview but products/users are still using it, we have added some experimental parameters that are not documented because we planed to remove them in the future (this quarter). We didn't want users to implement them and then introduce breaking changes.
We're currently updating the API documentation and cleaning some of those parameter, but it will still take a few days to be deployed in PROD.